index.md

```diff
@@ -22,13 +22,19 @@ visual and accessible presentations; to use the same documents in many
 different applications for different vendors; and to lengthen the time
 in which the documents are likely to be useable.
 
-While documents with declarative markup are often encoded in XML, XML
-vocabularies do not need to be declarative and declarative markup does
-not need to be in XML. The web is full of voices that champion clean,
-readable HTML documents styled with CSS. The academic fields of
-Digital Humanities and Corpus Linguistics, as well as companies from
-publishing houses to aircraft manufacturers, are developing more and
-more innovative uses for declarative markup.
+The time is ripe now to explain declarative markup: for a few years now, 
+non-declarative technologies have been developed and over-promised and their 
+limits have started to show.
+
+While documents with declarative markup are often encoded in XML, XML 
+vocabularies do not need to be declarative and declarative markup does 
+not need to be in XML. The web is full of voices that champion clean, 
+readable HTML documents styled with CSS. XML is a technology used by millions. 
+The academic fields of Digital Humanities and Corpus Linguistics, as well as 
+companies from publishing houses, manufacturers, insurance firms, healthcare 
```

Comment:
_I would replace the rest of the paragraph with this slightly edited version:_

organizations including publishing houses, manufacturers, insurance firms, healthcare 
providers, legal departments, ecommerce and government departments - any institutions, indeed, 
requiring open data policies or the long term preservation of information - 
have as much need as ever for declarative markup.